test/gurka/gurka.h

```diff
@@ -786,14 +200,16 @@ valhalla::Api locate(const map& map,
  * @return A JSON document created from serialized raw_result. Caller should
  * call HasParseError() on the returned document to verify its validity.
  */
-rapidjson::Document convert_to_json(valhalla::Api& raw_result, valhalla::Options_Format format) {
-  raw_result.mutable_options()->set_format(format);
+rapidjson::Document convert_to_json(valhalla::Api& raw_result, valhalla::Options_Format format);
 
-  std::string json = tyr::serializeDirections(raw_result);
-  rapidjson::Document result;
-  result.Parse(json.c_str());
-  return result;
-}
+// TODO: bidirectional edges overlap shapes so its hard to visualize them
+/**
+ * Dumps the gurka map to geojson including edges and nodes. Properties will be street names and node
+ * names. Edge ids are also included
+ * @param graph    the gurka map
+ * @return geojson string
+ */
+std::string dump_geojson_graph(const map& graph);
```

Comment:
i added this new function to dump geojson of a graph from the information in the gurka::map. right now the information is sparse but over time we can add info to the properties. the idea is that we have a way to see what is in the graphs taht we create. i  understand the ascii is supposed to do that but frankly sometimes its very hard to reason about what edges are where in space. further more the properties allow us to see if the data is getting set on the edges correctly.